### DISCUSSION TITLE
SDK: update an item and one of its O2M relation

### DISCUSSION THREAD

**Discussion by artecoop** [directus/directus#16701] [Archive]
Hey Everybody!

I have a question: using the js-sdk, is it possibile to update an item field and a related data? 
Take this example:
Imagine having a collection named `People` like this
- id: autogenerated
- name: string required
- addresses: O2M -> Address

and the related Address collection
- id: autogenerated
- address1: string required
- peopleId: automatically generated M2O field -> People

is it possible to update in a single shot a People record AND one of the related Address?

something like

```js
directus.items("People").updateOne(1, {
    name: "Prometeus",
    addresses: {
        id: 26,
        address1: 'Olimpus Road'
    }
});
```

Thanks in advance!

**Comment by rijkvanzanten** [comment]
Yup! The SDK works the same as the API, so you can use the same relational nested structure https://docs.directus.io/reference/introduction.html#relational-data
